components/counter/Counter.js

```diff
@@ -0,0 +1,54 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import omit from 'lodash.omit';
+import theme from './theme.css';
+
+class Counter extends PureComponent {
+  static propTypes = {
+    children: PropTypes.node,
+    className: PropTypes.string,
+    color: PropTypes.oneOf([ 'neutral', 'ruby' ]),
+    size: PropTypes.oneOf([ 'small', 'medium' ]),
+  };
+
+  static defaultProps = {
+    color: 'neutral',
+    size: 'medium',
+  };
+
+  render () {
+    const {
+      children,
+      className,
+      color,
+      size,
+      ...others
+    } = this.props;
+
+    const classes = cx(
+      theme.counter,
+      theme.monospaced,
```

Comment:
Seems like a leftover? The class isn't doing anything as far as I can see (although it actually has to be monospaced).